src/renderers/renderhelpers.cpp

```diff
@@ -332,13 +332,9 @@ void RenderEscapeCodes(const QByteArray &input, const QTextCharFormat& format, Q
         const auto currentCharacter = *it;;
         if (currentCharacter == escapeString)
         {
-            it++;
-            const auto escSequence = *it;
+            const auto escSequence = *++it;
```

Comment:
There are actually done on purpose as i don't like prefix/postfix operators. I'd rather keep the current state